extensions/kafka-eight-simpleConsumer/src/main/resources/META-INF/services/io.druid.initialization.DruidModule

```diff
@@ -0,0 +1 @@
+io.druid.firehose.kafka.KafkaEightSimpleConsumerDruidModule
```

Comment:
https://github.com/druid-io/druid/blob/master/extensions/namespace-lookup/src/main/resources/META-INF/services/io.druid.initialization.DruidModule

That one seems to work.